.travis.yml

```diff
@@ -194,6 +194,12 @@ matrix:
         - DESC="build with OracleJDK9"
         - CMD="mvn -e package -Passembly && mvn -e site -Dlinkcheck.skip=true"
 
+    # OracleJDK9 build
+    - jdk: oraclejdk9
+      env:
+        - DESC="build with OracleJDK9"
```

Comment:
Can we make description unique? it is the same as the previous one.

Reply:
fixed